README.md

```diff
@@ -2,16 +2,17 @@
 
 A Rust Crux Client crate/lib. For now, this crate intends to support 2 ways to interact with Crux:
 
-- [x] Via `Docker` with a [`crux-standalone`](https://opencrux.com/docs#config-docker) version [docker-hub](https://hub.docker.com/r/juxt/crux-standalone). Current Docker image `juxt/crux-standalone:20.07-1.10.0`.
-- [x] Via [`HTTP`](https://opencrux.com/docs#config-http) using the [`REST API`](https://opencrux.com/docs#restapi).
+- [x] Via `Docker` with a [`crux-standalone`](https://opencrux.com/reference/building.html#_docker) version [docker-hub](https://hub.docker.com/r/juxt/crux-standalone). Current Docker image `juxt/crux-standalone:20.07-1.10.0`.
+- [x] Via [`HTTP`](https://opencrux.com/reference/http.html#start-http-server) using the [`HTTP API`](https://opencrux.com/reference/http.html#http-api).
 - [x] Async support.
 - [ ] Clojure.api. (To be evaluated.)
 - [ ] FFI. (To be evaluated.)
 
 > Other solutions may be added after the first release.
 
-* For information on Crux and how to use it, please follow the link to [opencrux](https://opencrux.com/docs#restapi).
-* [**Crux FAQ**](https://opencrux.com/docs#faqs)
+* For information on Crux and how to use it, please follow the link to [opencrux](https://opencrux.com/reference/http.html#http-api).
```

Comment:
Should this link really point to the HTTP part? Or should it be more generic? :thinking:

Reply:
removed as it already has the getting started 69641e5